### PR TITLE
[REF] auto-fix conventions

### PR DESCRIPTION
**Pull request by moylop260**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Green :+1:

File: invoice_line_description/invoice.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def product_id_change(
         self, cr, uid, ids, product_id, uom_id, qty=0,
         name='', type='out_invoice', partner_id=False, fposition_id=False,
         price_unit=False, currency_id=False, context=None, company_id=None
-    ):
+    ):  # noqa
```
Comment (moylop260) [inline invoice_line_description/invoice.py:32]: Why noqa is required?

Comment (pedrobaeza) [inline invoice_line_description/invoice.py:32]: Because keyword argument `type` can't be changed.

**Comment by eLBati** [comment]
:+1:

File: invoice_line_description/invoice.py (hunk 1 above)
Comment (bwrsandman) [inline invoice_line_description/invoice.py:32]: @pedrobaeza then add that to the comment.

File: sale_order_partial_invoice/sale.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def field_qty_invoiced(self, cr, uid, ids, fields, arg, context):
                     res[line.id] += invoice_line.quantity  # XXX uom !
         return res
 
-    def field_qty_delivered(self, cr, uid, ids, fields, arg, context):
+    def field_qty_delivered(self, cr, uid, ids, field_list, arg, context):
```
Comment (bwrsandman) [inline sale_order_partial_invoice/sale.py:66]: :-1:

File: sale_order_partial_invoice/sale.py [hunk 3]
```diff
@@ -55,7 +55,7 @@
 class SaleOrderLine(orm.Model):
     _inherit = 'sale.order.line'
 
-    def field_qty_invoiced(self, cr, uid, ids, fields, arg, context):
+    def field_qty_invoiced(self, cr, uid, ids, field_list, arg, context):
```
Comment (bwrsandman) [inline sale_order_partial_invoice/sale.py:58]: :-1:

**Comment by bwrsandman** [comment]
This might need to be reverted. See https://github.com/OCA/maintainer-quality-tools/issues/141

**Comment by pedrobaeza** [comment]
I haven't seen any case where that renaming has any side effect. Have you found any?

**Comment by bwrsandman** [comment]
@gurneyalex ?

**Pull request merged**